### ISSUE TITLE
Monaco editor doesn't load when airgapped

### ISSUE THREAD

**Issue by philrz** [brimdata/zui#2915]
## tl;dr

When running airgapped (i.e., no Internet connection) the editor fails to load. There's Monaco-related errors in DevTools and other errors that don't show up if I repeat the same steps while Internet-connected.

## Details

Repro is with GA Zui [v1.4.1](https://github.com/brimdata/zui/releases/tag/v1.4.1). I bumped into this while trying to reproduce a problem reported in a [community Slack thread](https://brimdata.slack.com/archives/CTT24051C/p1701324006460239).

In the attached video I'm launching Zui fresh after having just installed it and while having no working Internet connectivity.

While the mere launch of the app seems healthy, I do see some errors in DevTools right out of the gate, and I don't see those same errors if I launch when Internet-connected. They may be unrelated to the editor problem, but I figured I'd point them out regardless if we want to better understand their origin and maybe catch/silence these in some way.

The more concerning problem is after I load some data and the editor area is populated with text that just says "Loading..." The DevTools console shows errors related to Monaco, among others.

https://github.com/brimdata/zui/assets/5934157/a5cc39c1-4ab8-40e8-8722-d3beafae2422

If there's some reason why we simply need Internet connectivity for the app to work, we'd certainly be free to set that expectation with users, but ideally we'd want to fail gracefully in a situation like this so the user is informed what's wrong.

**Comment by SoftTools59654** [comment]
I had this problem in a Windows 10 virtual machine

The virtual machine did not have access to the Internet and the data is not loaded

So the problem was not with the virtual machine, the problem was with the lack of internet in the virtual machine

**Comment by SoftTools59654** [comment]
I still have this problem in a system without internet, the latest version of zui insiders has not solved this problem?

**Comment by philrz** [comment]
@SoftTools59654: That is correct. The problem is still being worked on.

**Comment by philrz** [comment]
@SoftTools59654: Just a heads-up that we're about to put out a GA Zui release v1.5.0 but the fix for this issue is unfortunately is not yet complete. We hope to have a fix in a Zui Insiders release soon after. Thanks for your patience.

**Comment by SoftTools59654** [comment]
Is there a solution to this problem?
I installed the latest versions and the problem is still there

**Comment by philrz** [comment]
@SoftTools59654: Correct, the problem has not been fixed yet. We expect to have a fix for it soon. I'll update here when the fix is verified.

**Comment by philrz** [comment]
@SoftTools59654: FYI, the latest GA Zui release [v1.7.0](https://github.com/brimdata/zui/releases/tag/v1.7.0) that was just posted includes this fix.
